io.github.Tahoma2D.yaml

```
@@ -0,0 +1,180 @@
+app-id: io.github.Tahoma2D
+branch: stable
+runtime: org.kde.Platform
+runtime-version: 5.10
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.gfortran-62
+build-options:
+  append-path: /usr/lib/sdk/gfortran-62/bin
+command: tahoma2d
+finish-args:
+  - --socket=wayland
+  - --share=ipc
+  - --socket=x11
```

```suggestion
  - --socket=wayland
  - --share=ipc
  - --socket=fallback-x11
```

assuming it actually supports wayland